Sources/MediaPublicTracker.swift

```diff
@@ -160,7 +160,8 @@ class MediaPublicTracker: MediaTracker {
             }
 
             let currentTs = self.getCurrentTimeStamp()
-            if (currentTs - self.lastEventTs) > self.EVENT_TIMEOUT_MS {
+            let tsDelta = Int(currentTs - self.lastEventTs)
+            if tsDelta > self.EVENT_TIMEOUT_MS {
```

Comment:
Same comment, if you change the type of `EVENT_TIMEOUT_MS` to TimeInterval then you wouldn't need to cast `tsDelta` to `Int`.